static/js/lib/api_test.js

```diff
@@ -80,11 +79,7 @@ describe('api', function() {
 
     it('patches a user profile', () => {
       fetchJSONStub.returns(Promise.resolve(USER_PROFILE_RESPONSE));
-      fetchMock.mock('/api/v0/profiles/jane/', (url, opts) => {
-        assert.deepEqual(JSON.parse(opts.body), USER_PROFILE_RESPONSE);
-        return { status: 200 };
-      });
-      return patchUserProfile('jane', USER_PROFILE_RESPONSE).then(returnedProfile => {
+      patchUserProfile('jane', USER_PROFILE_RESPONSE).then(returnedProfile => {
```

Comment:
Ah, looks like you moved these `fetchMock` assertions into some more generalized tests down below - makes sense.